test/compiler-test.js

```diff
@@ -127,7 +127,7 @@ tests = {
     }
   },
   'a\\bc': {
-    abc: {
```

Comment:
mask is `a\bc` which tries to escape `b` character, which resolves into `abc` not `a\bc`.